### PR TITLE
Update changelog

### PR DESCRIPTION
**Pull request by NilashishC**
Signed-off-by: NilashishC <nilashishchakraborty8@gmail.com>

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
changelogs/

### CODE REVIEW

File: galaxy.yml [hunk 1]
```diff
@@ -8,5 +8,4 @@ name: nxos
 namespace: cisco
 readme: README.md
 repository: https://github.com/ansible-collections/nxos
-version: 0.0.1
```
Comment (Qalthos) [inline galaxy.yml:11]: I think you need to retain the version key for galaxy? https://github.com/ansible-collections/ansible.netcommon/blob/ff06176c598d9b187722799dce3daaf911792c39/galaxy.yml#L10

Comment (NilashishC) [reply galaxy.yml:11]: I brought this up in the meeting yesterday and I was told we can remove it. Also, nxos and netcommon seem to be the only collections that have this.

**Comment by NilashishC** [comment]
recheck

**Comment by NilashishC** [comment]
recheck

**Comment by NilashishC** [comment]
recheck

**Pull request merged**